csharp/ql/src/experimental/Security Features/JsonWebTokenHandler/delegated-security-validations-always-return-true.ql

```diff
@@ -0,0 +1,21 @@
+/**
+ * @name Delegated security sensitive validations for JsonWebTokenHandler always return true, medium precision
+ * @description Security sensitive validations for `JsonWebTokenHandler` are being delegated to a function that seems to always return true.
+ *   Higher precision version checks for exception throws, so less false positives are expected.
+ * @kind problem
+ * @tags security
+ *       JsonWebTokenHandler
+ *       manual-verification-required
+ * @id cs/JsonWebTokenHandler/delegated-security-validations-always-return-true
```

Comment:
The failing test is because of
```
WARNING: The @id property should be a valid query identifier. (/home/runner/work/semmle-code/semmle-code/ql/csharp/ql/src/experimental/Security Features/JsonWebTokenHandler/delegated-security-validations-always-return-true.ql:1,1-12,4)
```
(we treat warnings as errors). You should be able to see the warnings as well when executing `codeql test` (and possibly also in VS code).

Reply:
There is no warning in VS code when I run the unit test.